src/trainer_pools.c

```diff
@@ -167,6 +167,10 @@ static u32 PickMonFromPool(const struct Trainer *trainer, u8 *poolIndexArray, u3
     //  If no mon has been found yet continue looking
     if (monIndex == POOL_SLOT_DISABLED)
         monIndex = pickFunctions.OtherFunction(trainer, poolIndexArray, partyIndex, monsCount, battleTypeFlags, rules);
+    //  If no mon still hasn't been found, return error and default to regular part generation
```

Comment:
```suggestion
    //  If a mon still hasn't been found, return error and default to regular part generation
```
Not super necessary, but I struggled a little to parse the sentence on first read. Maybe "return _an_ error" too? But I didn't stumble on that, so it's an ever weaker suggestion.